Makefile

```
@@ -30,6 +30,32 @@ help: Makefile
 clean:
 	@rm -rf ${OUTDIR}
 
+.download-helm-if-needed:
+	@if [ "$(shell which helm 2>/dev/null 1>&2 && helm version --short | grep "v3" || echo -n "")" == "" ]; then \
+	  mkdir -p "${OUTDIR}/helm-install" ;\
+	  if [ -x "${OUTDIR}/helm-install/helm" ]; then \
+	    echo "You do not have helm installed in your PATH. Will use the one found here: ${OUTDIR}/helm-install/helm" ;\
+	  else \
+	    echo "You do not have helm installed in your PATH. The binary will be downloaded to ${OUTDIR}/helm-install/helm" ;\
+	    os=$$(uname -s | tr '[:upper:]' '[:lower:]') ;\
+	    arch="" ;\
+	    case $$(uname -m) in \
+	        i386)   arch="386" ;; \
+	        i686)   arch="386" ;; \
+	        x86_64) arch="amd64" ;; \
+	        arm)    dpkg --print-architecture | grep -q "arm64" && arch="arm64" || arch="arm" ;; \
+	    esac ;\
+	    mkdir -p "${OUTDIR}/helm-install" ;\
+		cd "${OUTDIR}/helm-install" ;\
```

you should not have to do this since it was already done on line 35 above. Also adjust indentation on the `cd` line.
```suggestion
	    cd "${OUTDIR}/helm-install" ;\
```

---

I include this in my diff down below when I fixed the issue with the helm executable not being found